run.py

```diff
@@ -0,0 +1,7 @@
+from Application import app
+import os
+
+app.secret_key = os.urandom(8)
+
+if __name__ == "__main__":
+    app.run(debug=True)
```

Comment:
you usually  avoid debug=True in production . just as a hint. For example to avoid that and have it in ur local configuration you could an ENV variable in PyCharm Run Configuration with the value FLASK_DEBUG=1